src/commands/moderation/prefixCommands/functions/setChannelDefaultVersion.ts

```diff
@@ -0,0 +1,108 @@
+import { ChatInputCommandInteraction, Colors, TextChannel, User } from 'discord.js';
+import { constantsConfig, getConn, PrefixCommandVersion, PrefixCommandChannelDefaultVersion, Logger, makeEmbed, loadSinglePrefixCommandChannelDefaultVersionToCache } from '../../../../lib';
+
+const noConnEmbed = makeEmbed({
+    title: 'Prefix Commands - Set Default Channel Version - No Connection',
+    description: 'Could not connect to the database. Unable to set the channel default version.',
+    color: Colors.Red,
+});
+
+const noVersionEmbed = (channel: string) => makeEmbed({
+    title: 'Prefix Commands - Show Default Channel Version - No Version',
+    description: `Failed to show default channel version for channel ${channel} as the configured version does not exist.`,
+    color: Colors.Red,
+});
+
+const failedEmbed = (channel: string) => makeEmbed({
+    title: 'Prefix Commands - Set Default Channel Version - Failed',
+    description: `Failed to set the channel default version for channel ${channel}.`,
+    color: Colors.Red,
+});
+
+const successEmbed = (channel: string, version: string, emoji: string) => makeEmbed({
+    title: `Prefix Command Channel Default version set for channel ${channel} to version ${version} ${emoji}.`,
+    color: Colors.Green,
+});
+
+const modLogEmbed = (moderator: User, channel: string, version: string, emoji: string) => makeEmbed({
+    title: 'Prefix channel default version set',
+    fields: [
+        {
+            name: 'Channel',
+            value: channel,
+        },
+        {
+            name: 'Version',
+            value: `${version} - ${emoji}`,
+        },
+        {
+            name: 'Moderator',
+            value: `${moderator}`,
+        },
+    ],
+    color: Colors.Green,
+});
+
+const noModLogs = makeEmbed({
+    title: 'Prefix Commands - Set Default Channel Version - No Mod Log',
+    description: 'I can\'t find the mod logs channel. Please check the channel still exists.',
+    color: Colors.Red,
+});
+
+export async function handleSetPrefixCommandChannelDefaultVersion(interaction: ChatInputCommandInteraction<'cached'>) {
+    await interaction.deferReply({ ephemeral: true });
+
+    const conn = getConn();
+    if (!conn) {
+        await interaction.followUp({ embeds: [noConnEmbed], ephemeral: true });
+        return;
+    }
+
+    const channel = interaction.options.getChannel('channel')!;
+    const version = interaction.options.getString('version')!;
+    const moderator = interaction.user;
+
+    //Check if the mod logs channel exists
+    const modLogsChannel = interaction.guild.channels.resolve(constantsConfig.channels.MOD_LOGS) as TextChannel;
+    if (!modLogsChannel) {
```

Comment:
```suggestion
    let modLogsChannel = interaction.guild.channels.resolve(constantsConfig.channels.MOD_LOGS);
    if (!modLogsChannel || !modLogsChannel.isTextBased()) {
        modLogsChannel = null;
```
Might be safer to actually check if the channel is a text channel? Especially if we want to offer the codebase to other projects. Then, if it's not a text channel set it to null to prevent accidental attempts to send into a non-text channel.